docs/api/devguide/spatialsearch.md

```diff
@@ -0,0 +1,204 @@
+# Spatial Search for Features
+
+This section describes how to search for specific features in your space which are intersecting with a provided geometry.
+
+## Possibilities to provide a geometry for a spatial search
```

Comment:
Your titles seem to go back and forth between sentence and title case. So for example, why isn't this "Possibilities to Provide a Geometry for a Spatial Search"? The next title seems to use titlecase but then the one after doesn't. Check to see what other docs do and follow that.